app/Console/cake.php

```diff
@@ -18,29 +18,29 @@
  */
 
 if (!defined('DS')) {
-	define('DS', DIRECTORY_SEPARATOR);
+    define('DS', DIRECTORY_SEPARATOR);
 }
 
 $dispatcher = 'Cake' . DS . 'Console' . DS . 'ShellDispatcher.php';
 
 if (function_exists('ini_set')) {
-	$root = dirname(dirname(dirname(__FILE__)));
-	$appDir = basename(dirname(dirname(__FILE__)));
-	$install = $root . DS . 'lib';
-	$composerInstall = $root . DS . $appDir . DS . 'Vendor' . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
+    $root = dirname(dirname(dirname(__FILE__)));
+    $appDir = basename(dirname(dirname(__FILE__)));
+    $install = $root . DS . 'lib';
+    $composerInstall = $root . DS . $appDir . DS . 'Vendor' . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
 
-	// the following lines differ from its sibling
-	// /lib/Cake/Console/Templates/skel/Console/cake.php
-	if (file_exists($composerInstall . DS . $dispatcher)) {
-		$install = $composerInstall;
-	}
+    // the following lines differ from its sibling
+    // /lib/Cake/Console/Templates/skel/Console/cake.php
+    if (file_exists($composerInstall . DS . $dispatcher)) {
+        $install = $composerInstall;
+    }
 
-	ini_set('include_path', $install . PATH_SEPARATOR . ini_get('include_path'));
-	unset($root, $appDir, $install, $composerInstall);
+    ini_set('include_path', $install . PATH_SEPARATOR . ini_get('include_path'));
```

Comment:
__[staging][code_sniffer]__
* Concat operator must not be surrounded by spaces
* Concat operator must not be surrounded by spaces